src/client/game.cpp

```diff
@@ -4089,8 +4089,8 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats, f32 dtime,
 		float old_brightness = sky->getBrightness();
 		direct_brightness = client->getEnv().getClientMap()
 				.getBackgroundBrightness(MYMIN(runData.fog_range * 1.2, 60 * BS),
-					daynight_ratio, (int)(old_brightness * 255.5), &sunlight_seen)
-				    / 255.0;
+						daynight_ratio, (int)(old_brightness * 255.5), &sunlight_seen)
+				/ 255.0;
```

Comment:
```suggestion
					daynight_ratio, (int)(old_brightness * 255.5), &sunlight_seen)
					/ 255.0;
```
You removed 4 spaces, but placed the tab at the wrong line

Reply:
I have indented 4092 by two tabs because it is part of the method getBackgroundBrightness calling from the previous line.
and line 4093 is not part of it. So I continue in the same indent as 4091 did.

If I understand well from our code style guide, if a command is not finished on one line, the next line should be two tabs indented from the line where the command begins.

Reply:
Makes sense, I thought, you only replaced the spaces with a tab.